apps/nextjs-website/src/components/atoms/ProductHeader/ProductHeader.tsx

```diff
@@ -20,10 +20,11 @@ const ProductHeader: FC<ProductHeaderProps> = ({ product, path }) => {
   return (
     <Box
       sx={{
-        position: 'sticky',
+        position: 'fixed',
         top: scrollUp ? SITE_HEADER_HEIGHT : 0,
-        zIndex: 101,
+        zIndex: scrollUp ? 90 : 250,
         transition: 'all 0.5s linear',
```

Comment:
I think `transition: all` cause width resizing problem:
[Screencast from 16-01-2024 16:56:55.webm](https://github.com/pagopa/developer-portal/assets/24475319/7deff611-c30d-4b17-86bf-092e0ea4a48f)
```suggestion
        transition: 'top 0.5s linear',
```